.changeset/thick-bags-grab.md

```diff
@@ -1,5 +1,5 @@
 ---
-"@xata.io/cli": patch
+'@xata.io/cli': patch
```

Comment:
ha, I think prettier did that when I was looking at where changesets are. But should be fine

Reply:
Ah, tracking PR that's why https://github.com/xataio/client-ts/pull/1547